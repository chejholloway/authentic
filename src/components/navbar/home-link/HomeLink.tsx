import React from "react";
import { Link } from "react-router-dom";

const HomeLink: React.FC = () => {
  return (
    <div className="flex items-center space-x-5">
      <Link
        to="/"
        className="text-black mr-4 hover:text-gray-400 transition duration-300"
      >
        Home
      </Link>
    </div>
  );
};

export default HomeLink;
