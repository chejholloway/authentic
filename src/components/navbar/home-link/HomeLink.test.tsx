import { render } from '@testing-library/react';
import React from 'react';
import { Link } from "react-router-dom";
import HomeLink from "./HomeLink";

jest.mock("react-router-dom");

describe('<HomeLink>', () => {
  it('should render component', () => {
    const { container } = render(<HomeLink />);

    expect(container).toMatchSnapshot();
  });
});
