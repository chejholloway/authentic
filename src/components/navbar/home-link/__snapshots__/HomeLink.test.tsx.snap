// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeLink> should render component 1`] = `
<div>
  <div
    class="flex items-center space-x-5"
  />
</div>
`;
